lib/view.dart

```diff
@@ -104,9 +104,40 @@ void clearContentTab() {
   content.children.clear();
 }
 
-void renderAnalyseTab() {
-  clearContentTab();
-  content.append(html.DivElement()..innerText = 'Analyse');
+void renderAnalysisTabRadio(List<String> labels) {
```

Comment:
I'm not sure we need to surface how the analysis tabs are displayed - perhaps something like `renderAnalysisTabBar` or `renderAnalysisTabOptions` or even just `renderAnalysisTabs`?

Reply:
Fixed to `renderAnalysisTabs`